src/main/java/duke/storage/Storage.java

```diff
@@ -0,0 +1,53 @@
+package duke.storage;
+
+import duke.task.Task;
+
+import java.util.ArrayList;
+
+import java.io.FileInputStream;
+import java.io.FileOutputStream;
+import java.io.IOException;
+import java.io.ObjectInputStream;
+import java.io.ObjectOutputStream;
+
+
+public class Storage {
+
+    private static String filePath = "src/main/java/TaskDatabase.ser";
+
+    public Storage(String path) {
+        filePath = path;
+    }
+
+
+    //@@author zzhikai-reused
+    //Reused from https://samderlust.com/dev-blog/java/write-read-arraylist-object-file-java
+    // with minor modification
```

Comment:
Was there any particular reason why you used his code instead of something simpler like `FileReader`/`FileWriter`? 🤔